production/ksonnet/loki/bloom-gateway.libsonnet

```diff
@@ -0,0 +1,159 @@
+{
+  local k = import 'ksonnet-util/kausal.libsonnet',
+  local container = k.core.v1.container,
+  local containerPort = k.core.v1.containerPort,
+  local pvc = k.core.v1.persistentVolumeClaim,
+  local service = k.core.v1.service,
+  local statefulSet = k.apps.v1.statefulSet,
+  local volume = k.core.v1.volume,
+  local volumeMount = k.core.v1.volumeMount,
+
+  local name = 'bloom-gateway',
+
+  _config+:: {
+    bloom_gateway+: {
+      use_local_ssd: false,
+      // PVC config
+      pvc_size: '512Gi',
+      pvc_class: 'fast',
+      // local SSD config
+      hostpath: '/mnt/disks/ssd0',
+      node_selector: { 'has-local-ssd': 'true' },
+      tolerations: [
+        k.core.v1.toleration.withKey('type') +
+        k.core.v1.toleration.withOperator('Equal') +
+        k.core.v1.toleration.withValue('local-ssd') +
+        k.core.v1.toleration.withEffect('NoSchedule'),
+      ],
```

Comment:
`node_selector` and `tolerations` are usually specific to the K8s setup, so they should be provided by the user.